1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,12 +21,13 @@ let basketballTeam = {
 */
 
 // write code here
-
-
+basketballTeam.topPlayers
+  .sort()
+  .forEach((topPlayers) => console.log(topPlayers));
```

Comment:
How the variables are named, can really help with understanding what's going on. In this case, the `forEach` function receives one array item each time it's called, so `topPlayer` is a better name for the parameter.